### PR TITLE
Update NLS translations

### PR DESCRIPTION
**Pull request by pshipton**
Depends on https://github.com/eclipse-openj9/openj9/pull/18446

### CODE REVIEW

File: runtime/nls/j9vm/j9vm_fr.nls [hunk 1]
```diff
@@ -279,7 +279,7 @@ J9NLS_VM_CHECK_JAVA_HOME.user_response=Check that JAVA_HOME environment variable
 # This message will be followed by the exception's class name and
 # (if non-null) the exception's detail message
 # e.g. Exception in thread "main" java/lang/StackOverflowError: An overflow occurred
-J9NLS_VM_STACK_TRACE_EXCEPTION_IN=Exception dans l'unit\u00e9 d'ex\u00e9cution "%.*s"\u0020
+J9NLS_VM_STACK_TRACE_EXCEPTION_IN=Exception dans l'unit\u00e9 d'ex\u00e9cution "%.*s"
```
Comment (keithc-ca) [inline runtime/nls/j9vm/j9vm_fr.nls:282]: I don't think we want this sort of change.

**Review by keithc-ca — CHANGES_REQUESTED** [review]

File: runtime/nls/j9vm/j9vm_fr.nls (hunk 1 above)
Comment (pshipton) [reply runtime/nls/j9vm/j9vm_fr.nls:282]: Why is that? It's what the automated translation process does, I don't want to have to change them back every time. The downside would be that it fails the line ending check.

**Comment by pshipton** [comment]
jenkins line endings check

File: runtime/nls/j9vm/j9vm_fr.nls (hunk 1 above)
Comment (pshipton) [reply runtime/nls/j9vm/j9vm_fr.nls:282]: The check does indeed fail, but all the failures are for J9NLS_VM_STACK_TRACE_EXCEPTION_IN

Comment (keithc-ca) [reply runtime/nls/j9vm/j9vm_fr.nls:282]: We should remove that part of the automation, or maybe better, arrange that messages don't need the trailing space.

Comment (pshipton) [reply runtime/nls/j9vm/j9vm_fr.nls:282]: I have no control over the actual automation, although I could do my own post processing steps. 
Created https://github.com/eclipse-openj9/openj9/pull/18438

**Comment by keithc-ca** [comment]
Spot-checking, it appears as if this is completely included in #18446. Is that intentional, meaning we'll just close this after #18446 is merged?

**Comment by pshipton** [comment]
The commit in https://github.com/eclipse-openj9/openj9/pull/18446 was a cherry pick of the commit here.

**Pull request closed without merging**